package.json

```diff
@@ -10,7 +10,7 @@
     "doctoc": "doctoc --maxlevel 3 README.md",
     "jsdoc:widget": "node ./scripts/doc/gen-widget-doc.js",
     "gh-pages": "./scripts/gh-pages.sh",
-    "lint": "eslint . --quiet --no-color && sass-lint --verbose",
+    "lint": "eslint ./components ./decorators ./lib ./widgets --quiet --no-color && sass-lint --verbose",
```

Comment:
I was aiming at excluding the webpack config and the `./scripts`.

Tests should be targeted as well, but you're right that something is not working as expected here. Will have another look.